### PR TITLE
Add Polar chart (#284, #259)

### LINKED ISSUE #284: Create a polar charts section
**Description**

See [polar chart content](https://docs.google.com/document/d/1gEOBO8QEONQbfBawE76EMvzkZGuD_58d).

We will want to find relevant datasets, based on cyclic events.

### PR DESCRIPTION
**Pull request by drake-stamos**
Addresses #284 and #259.

### CODE REVIEW

**Review by FredLL-Avaiga — COMMENTED** [review]
I guess there could be less space lost in the images ?
Do you plan to produce dark theme screenshots also ?
Are the md files planned in another PR ?

**Comment by haotran-avaiga** [comment]
Hi Drake, 

There are some useful comments in my PR, I think you can take a look and apply to your PR.
https://github.com/Avaiga/taipy-doc/pull/292

File: docs/manuals/gui/viselements/charts/polar-chart.md [hunk 1]
```diff
@@ -0,0 +1,586 @@
+# Polar Charts
+
+Polar Chart is a common variation of circular graphs. It is useful when relationships between data points can be visualized most easily in terms of radiuses and angles.
+
+In Polar Charts, a series is represented by a closed curve that connects points in the polar coordinate system. Each data point is determined by the distance from the pole (the radial coordinate) and the angle from the fixed direction (the angular coordinate).
+
+## Polar Charts Key Parameters <a id="table"></a>
+
+Explanation of parameters and options relevant to radar charts can be found [here](#table). Full details for the Taipy Chart control can be found in the Chart Control section of the [Taipy GUI User Manual](https://docs.taipy.io/en/latest/manuals/gui/viselements/chart/)
```
Comment (dinhlongviolin1) [inline docs/manuals/gui/viselements/charts/polar-chart.md]:
```suggestion
Explanation of parameters and options relevant to polar charts can be found [here](#table). Full details for the Taipy Chart control can be found in the Chart Control section of the [Taipy GUI User Manual](https://docs.taipy.io/en/latest/manuals/gui/viselements/chart/)
```

**Review by dinhlongviolin1 — APPROVED** [review]
Examples look good to me

**Review by FredLL-Avaiga — APPROVED** [review]

**Pull request merged**